assets/scss/checkout.scss

```diff
@@ -1,3 +1,85 @@
 // =============================================================================
 // Stencil Checkout  - Customize the checkout experience
 // =============================================================================
+
+// Main content
+.ng-checkout-container {
+    background-color: stencilColor("uco_main_backgroundColor");
+}
+
+// Cart
+.cart {
+    background-color: stencilColor("uco_cart_backgroundColor");
+}
+
+// Typography
+a {
+    color: stencilColor("uco_links_color");
+}
+
+.button {
+    background-color: stencilColor("uco_buttons_color");
+}
+
+.form-field--error {
+
+    .form-input,
+    .form-select,
+    .form-checkbox + .form-label::before,
+    .form-radio + .form-label::before,
+    .form-prefixPostfix-label {
+        border-color: stencilColor("uco_errors_color");
+    }
+
+    .form-field-error,
+    .form-inlineMessage {
+        color: stencilColor("uco_errors_color");
+    }
+
+    // Depth need to style svg with validation state
+    // scss-lint:disable SelectorDepth
+    .form-input-indicator > svg {
+        fill: stencilColor("uco_errors_color");
+    }
+    // scss-lint:enable SelectorDepth
+}
+
+.form-legend,
+.form-body,
+.shippingOptions-panel {
+    background-color: stencilString("uco_form-field_backgroundColor");
+}
+
+// Font Overrides
+$fontFamily-headings:           stencilFontFamily("uco_headings_font"), Arial, Helvetica, sans-serif;
+$fontFamily-sans:               stencilFontFamily("uco_body_font"), Arial, Helvetica, sans-serif;
+
+// Font Headings
+h2 {
+    font-family: $fontFamily-headings;
+}
+
+.orderedList-item {
+    &::before {
+        font-family: $fontFamily-headings;
+    }
+}
+
+// Font Body
+body {
+    font-family: $fontFamily-sans;
+}
```

Comment:
This can probably go to typography too. I'm just suggesting that because you're not using classes to style this stuff, so it might be good to group it together early on for structure and readability.